docs/pages/push-notifications/sending-notifications-custom.mdx

```diff
@@ -122,6 +122,42 @@ Your FCM server key can be found by making sure you've followed the [configurati
 
 Communicating with APNs is a little more complicated than with FCM. Some libraries wrap all of this functionality into one or two function calls such as [`node-apn`](https://github.com/node-apn/node-apn). However, in the examples below, a minimum set of libraries are used.
 
+### Client APNs entitlement
+
+The steps below will only work if your iOS app has the APNs entitlement. For apps using [CNG](/workflow/continuous-native-generation/),
+the Expo config needs to be modified in one of two ways:
+
+- **Recommended**: Add the `expo-notifications` library to your app, and ensure that its plugin appears in the `plugins` array of your [app config](/workflow/configuration/):
+
+```json app.json
+{
+  "expo": {
+    /* @hide ... */ /* @end */ "plugins": [/* @hide ... */ /* @end */ "expo-notifications"]
+  }
+}
+```
+
+- If you do not plan to use the `expo-notifications` library, then you should [add the `aps-environment` entitlement manually to the Expo configuration](/build-reference/ios-capabilities/#entitlements),
+  as in the example below:
+
+```json app.json
+{
+  "expo": {
+    /* @hide ... */ /* @end */
+    "ios": {
+      /* @hide ... */ /* @end */
+      "entitlements": {
+        "aps-environment": "development"
```

Comment:
Is it always development, regardless of the build type?

Reply:
I believe that xcode will automatically handle this (switch to `production`) when building for release